docs/products/postgresql/howto/identify-pg-slow-queries.md

```diff
@@ -102,7 +91,7 @@ wal_fpi             | bigint           |           |          |
 wal_bytes           | numeric          |           |          |
 ```
 
-On older PostgreSQL versions you might find different column names (for example,
+On older PostgreSQL versions, you might find different column names (for example,
```

Comment:
We are aging PostgreSQL 12. We blocked the new service creation and the overall support will end on October 14th, 2024. We might choose to remove any references to version 12 and earlier now or we might do it in October. I suggest we do it at this time.

Reference: https://aiven.io/docs/platform/reference/eol-for-major-versions